reading feedback.py

```diff
@@ -0,0 +1,84 @@
+import openai

```

Comment:
For the pull request, please provide the description so that a reviewer can know from the high level what are the changes.